include/dfu/pcd.h

```diff
@@ -53,10 +53,19 @@ extern "C" {
 
 enum pcd_status {
 	PCD_STATUS_COPY = 0,
-	PCD_STATUS_COPY_DONE = 1,
-	PCD_STATUS_COPY_FAILED = 2
+	PCD_STATUS_DONE = 1,
+	PCD_STATUS_FAILED = 2,
+	PCD_STATUS_READ_VERSION = 3,
 };
 
+/** @brief Sets up the PCD command structure and point the data buffer to version
+ *
+ * @param version Pointer to a doubleword sized buffer in RAM.
```

Comment:
Can we not use things like doubleword and to highlight why to avoid them is word on armv7 is 32 bits, so "doubleword" here would be 64-bits

Reply:
Agreed above